docs/content/get-started/developers.mdx

```diff
@@ -5,18 +5,188 @@ nav_order: 20
+
+```bash
+cd path/to/project/root
+```
+
+Initialize your project to create a file called package.json. Once you have this file, you can use npm to install software (or packages) like NCIDS.
+
+```bash
+npm init
+```
+
+This initialization will start a series of prompts at the command line. Usually the defaults (which are noted in parentheses) are okay for a simple project. You can always edit these values later.
+
+In order to download the NCIDS, you will have to set up the @nciocpl organizational scope in npm to see the package. Create a `.npmrc` file in the root of your project containing the following:
+
+```bash
+@nciocpl:registry=https://npm.pkg.github.com
+```
+
+### Step 4: Install NCIDS
+
+Now, you can install NCIDS into your project from the command line with npm.
+
+At minimum, your project will require the `ncids-css` package. Install it via the command line:
+
+```bash
+npm install @nciocpl/ncids-css
+```
+
+In order to use the javascript components, download the `ncids-js` package separately. Install it via the command line:
+
+```bash
+npm install @nciocpl/ncids-js
+```
+
+`Don’t modify the source code. Now that you’ve installed the NCIDS source code, it is controlled by npm and could be rewritten at any time.`
+
+## Phase 2: Compile
+
+NCIDS Sass needs three things to compile properly:
+
+- Sass Module syntax: NCIDS requires a modern Sass compiler that can parse Sass Module syntax.
+- Autoprefixing: NCIDS requires Autoprefixing your CSS with a specific .browserslistrc.
+- Sass Load Paths: NCIDS requires Sass compilers use Load Paths that reference the /packages directory in the USWDS package
+
+### Autoprefixing
+
+The design system requires autoprefixing to work properly. Don't add vendor prefixes to your custom styles manually — it is more reliable to use autoprefixing.
+
+We use the following autoprefixer settings via .browserslistrc config:
+
+```js
+> 2%
+last 2 versions
+IE 11
+not dead
+```
+
+#### About Sass Load Paths
+
+`ncids-css` requires [Sass Load Paths](https://sass-lang.com/documentation/at-rules/use#load-paths) to compile for use in your project.
+
+Load paths must include a path to the `/packages` directory for NCIDS packages and `/uswds-packages` for USWDS packages. An example using Webpack might look like:
+
+```js
+{
+		loader: 'sass-loader',
+		options: {
+			sassOptions: {
+				includePaths: [
+					path.join(
+						__dirname,
+						'./node_modules/@nciocpl/ncids-css/packages'
+					),
+					path.join(
+						__dirname,
+						'./node_modules/@nciocpl/ncids-css/uswds-packages'
+					),
+				],
+			},
+		},
+},
+```
+
+### Import NCIDS styles to your project
+
+1. Create a sass stylesheet in your project.
+
+Import the required sass partials:
+
+```sass
+@use 'uswds-core' with (
+	$theme-image-path: '@nciocpl/ncids-css/uswds-img',
+);
+@forward 'uswds-global';
+```
+
+Next, add the entire NCIDS package library or just the packages you need:
+
+- Option 1: Import the entire ncids-css library.
```

Comment:
switch option 1 and 2 around, see comments about recommendations 

https://github.com/NCIOCPL/ncids-example-sites/blob/main/ncids-webpack/src/index.scss